lib/git_reflow/sandbox.rb

```diff
@@ -23,8 +31,10 @@ def run(command, options = {})
         output = %x{#{command}}
 
         if options[:blocking] == true && !$?.success?
-          abort "\"#{command}\" failed to run."
+          raise CommandError.new(output, "\"#{command}\" failed to run.") if options[:raise] == true
+          abort "\"#{command}\" failed to run." if options[:blocking] == true
         else
+          raise CommandError.new(output, "\"#{command}\" failed to run.") if options[:raise] == true && !$?.success?
```

Comment:
Consider handling this inside the top part of the conditional, changing it to look only for `!$?.success?`